package.json

```diff
@@ -289,6 +289,7 @@
 		"qs": "^6.9.1",
 		"react": "^16.12.0",
 		"react-dom": "^16.12.0",
+		"react-docgen-typescript-loader": "^3.7.2",
```

Comment:
Can we move these new deps to `packages/components/package.json`?

Reply:
Sure!

Reply:
Done in a1d3bb479ec1f0e4178956e0def059ecdbfb031a